internal/service/project/resource_project_test.go

```diff
@@ -95,10 +94,11 @@ func TestGetProjectPropsFromAPI(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			testObject := new(MockProjectService)
-			testObject.On("ListProjectTeams", mock.Anything, mock.Anything).Return(tc.teamRoleReponse)
-			testObject.On("ListProjectLimits", mock.Anything, mock.Anything).Return(tc.limitResponse)
-			testObject.On("GetProjectSettings", mock.Anything, mock.Anything).Return(tc.groupResponse)
+			testObject := mocksvc.NewGroupProjectService(t)
```

Comment:
knitpit: I like svc better than testObject but it's subjective

Reply:
I don't have a strong opinion on this, I will change it.